clang/docs/ReleaseNotes.rst

```diff
@@ -289,6 +289,11 @@ Bug Fixes to Compiler Builtins
 Bug Fixes to Attribute Support
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+- Clang now emits a warning instead of an error when using the one or two
+  argument form of GCC 11's ``__attribute__((malloc(deallocator)))``
+  or ``__attribute__((malloc(deallocator, ptr-index)))``
+  (`#51607 <https://github.com/llvm/llvm-project/issues/51607>`).
```

Comment:
```suggestion
  (`#51607 <https://github.com/llvm/llvm-project/issues/51607>`_).
```

Reply:
Sorry! That's my fault for being lazy and skipping build the docs! Fixed in 5f6f893c867e73a2a8e04e94a2fa6e2104b9b85e